Formula/req.rb

```diff
@@ -0,0 +1,32 @@
+class Req < Formula
+  desc "Simple and opinionated HTTP scripting language"
+  homepage "https://andrewpillar.com/programming/2022/02/26/req-an-http-scripting-language/"
```

Comment:
maybe use github page, https://github.com/andrewpillar/req, that url looks like an intro blog post?